lib/main_common.pm

```diff
@@ -1402,16 +1403,16 @@ sub load_extra_tests_y2uitest_gui {
         && is_desktop_installed()
         && !get_var("DUALBOOT")
         && !get_var("RESCUECD"));
-    loadtest 'yast2_gui/yast2_control_center';
-    loadtest "x11/yast2_lan_restart";
+    #loadtest 'yast2_gui/yast2_control_center';
```

Comment:
i suppose `#` should be removed, and was only for development

Reply:
comments have been deleted ;)

Reply:
You just removed all tests from `load_extra_tests_y2uitest_gui` except `yast2_bootloader`.